### ISSUE TITLE
PanelMenu nested subitems no longer get the expanded prop

### ISSUE THREAD

**Issue by digitalcraftco** [primefaces/primeng#15403]
### Describe the bug

In previous version 16.x.x 2nd & 3rd level nested sub menu items also had an expanded prop **in the model** when toggled open. In versions 17.x.x they are no longer getting the expanded prop. Only the 1st level parent items.

This makes it impossible to persist the expanded state of the panel menu beyond the first level. On a page refresh the menu will be reset to it's original state and user loses context.

```
{
    expanded: true,
    label: "Admin",
    items: [
         { 
            expanded: true,   < --- ! Not added here anymore. Same with deeper levels
            label: "Tasks",
            items: [...]
        },
        ...
    ]
 }
```

### Environment

Angular 18.0.1

### Reproducer
Toggle the menu a few items deep. The model will update below. Notice no nested items get an expanded attribute. Only top level parents. If you manually add expanded attribute to nested child (Documents) it won't update either. Notice how Files changes from false to true.

https://stackblitz.com/edit/s5kubx-5x7cem?file=src%2Fapp%2Fpanel-menu-basic-demo.html

### Angular version

18.0.1

### PrimeNG version

17.18.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20.9.0

### Steps to reproduce the behavior

Toggle a panel menu with 3rd or 4th level items. Inspect the model and you will see the nested parent items do not have the expanded: true property.

**Comment by tarockx** [comment]
Are there any news on this issue? I've been using PrimeNG 17.x.x and I can confirm there is no way to get the expanded state of a sub-item. At the moment I'm working around this by fetching the corresponding `<li>` element from the DOM and checking the `aria-expanded` attribute, but it would be good (and make the most sense, in my opinion) to have native support for this.

**Comment by flier268** [comment]
So do I

**Comment by digitalcraftco** [comment]
@cagataycivici @cetincakiroglu 4 month bump. Can someone triage this so I can decide if I need to try and PR a fix or if there are other reasons why this happened? Many thanks!

**Comment by ben-lewis-wtw** [comment]
What version of PrimeNG was this fixed in? As far as I can tell `expanded` is still not properly updated for nested nodes on [17.18.15](https://stackblitz.com/edit/s5kubx-wprt15hu?file=src%2Fapp%2Fpanel-menu-basic-demo.ts), but is fixed by [18.0.2](https://stackblitz.com/edit/stackblitz-starters-7aer2ycy?file=src%2Fapp%2Fapp.component.html).
